src/Symfony/Component/Cache/Adapter/RedisTagAwareAdapter.php

```diff
@@ -154,7 +157,7 @@ protected function doDeleteYieldTags(array $ids): iterable
         });
 
         foreach ($results as $id => $result) {
-            if ($result instanceof \RedisException || $result instanceof ErrorInterface) {
+            if ($result instanceof \RedisException || $result instanceof Exception || $result instanceof ErrorInterface) {
```

Comment:
Does it make sense to not import `Relay\Exception` for readability?

Reply:
I don't have preference on this, let's see if maintainers agree

Reply:
let's inline I agree, same as we did for `\Predis\ClientInterface`